tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,1164 @@
+{
```

Comment:
<p>The "Cluster some layers" use case should no longer be covered in the end-to-end example, but FYI for when you move it to the comprehensive guide.</p><p> </p><p>A lot of the functionality in <code>wrap_model_selectively</code>  can be covered via the <code>tf.keras.models.clone_model</code>  API.</p><p>See the "Quantize some layers" section of the <a href="https://github.com/tensorflow/model-optimization/pull/290" target="_blank">QAT comprehensive guide</a>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>